tests/integration/test_integration_watcher.py

```diff
@@ -1,20 +1,32 @@
 from fastapi import FastAPI
 from starlette.testclient import TestClient
 
+from antarest.core.tasks.model import TaskStatus
```

Comment:
Can you follow the [Integration Testing Guide with Pytest](https://github.com/AntaresSimulatorTeam/AntaREST/discussions/1712) to reorganize your test scenarios.